rholang/src/test/scala/coop/rchain/rholang/interpreter/accounting/CostAccountingPropertyTest.scala

```diff
@@ -105,7 +105,7 @@ object CostAccountingPropertyTest {
     mkRuntime[Task](prefix, 1024 * 1024).use { runtime =>
       for {
         _    <- runtime.cost.set(Cost.UNSAFE_MAX)
-        _    <- Runtime.injectEmptyRegistryRoot[Task](runtime.space, runtime.replaySpace)
+        _    <- Runtime.bootstrapRegistry[Task](runtime)
```

Comment:
I've got a hunch that registry is not needed for this test. Can you check this? (just delete this line :))

Reply:
Yep, you were right.

Reply:
Done